src/openrct2-ui/windows/Park.cpp

```diff
@@ -1446,6 +1446,22 @@ static void window_park_objective_invalidate(rct_window* w)
     window_park_anchor_border_widgets(w);
 }
 
+static void window_park_objective_bars_paint(
+    float percentage, int32_t x, int32_t y, rct_window* w, rct_drawpixelinfo* dpi, int32_t colour)
+{
+    if (percentage > 1)
+    {
+        percentage = 1;
+    }
+    int16_t barLength = static_cast<int16_t>((221 - 4) * percentage);
+
+    gfx_fill_rect_inset(dpi, x + 1, y + 1, x + 221 - 3, y + 9, w->colours[1], INSET_RECT_F_30);
```

Comment:
What is the `221` magic number?

Reply:
It just worked out the best for the size of the rectangle to fill the width of the park window.
I don't think the other code that used the function had a proper explanation for its size, too.

Reply:
I've looked at the function window_guest_stats_bars_paint in Guest.cpp. There are magic numbers in that code, too. It is used to paint the guest bars and that's the code that I used as example. I used my 221 like the 121 was used there.

Maybe, there's a solution for both functions. But I don't see it. Do you have an idea?

Reply:
I think it's alright to keep the magic numbers in this case.